components/bluetooth/lib.rs

```diff
@@ -693,323 +641,158 @@ impl BluetoothManager {
         }
     }
 
-    // https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattserver-getprimaryservice
-    // https://webbluetoothcg.github.io/web-bluetooth/#getgattchildren
-    fn get_primary_service(&mut self,
-                           device_id: String,
-                           uuid: String,
-                           sender: IpcSender<BluetoothResponseResult>) {
-        // Step 5.
-        if !self.device_is_cached(&device_id) {
-            return drop(sender.send(Err(BluetoothError::InvalidState)));
-        }
+    fn get_gatt_children(&mut self,
```

Comment:
Let's add the https://webbluetoothcg.github.io/web-bluetooth/#getgattchildren link back.